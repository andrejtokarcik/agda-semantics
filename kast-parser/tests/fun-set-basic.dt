  s = Set
; f = Lambda x s x
