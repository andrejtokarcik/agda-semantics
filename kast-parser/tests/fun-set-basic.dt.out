<T>
    <k>
        .K
    </k>
    <mgu>
        subst(.KList)
    </mgu>
    <tenv>
        f |-> 'Pi___(x,, Set 0,, Set 0)
        s |-> Set 1
    </tenv>
    <data>
        .Map
    </data>
    <fun>
        f |-> lambda ( closure ( x , x , s |-> Set 0 ) )
        s |-> Set 0
    </fun>
    <meta>
        .Map
    </meta>
</T>
